.github/workflows/bot.yml

```
@@ -22,7 +22,7 @@ on:
 
 jobs:
   pulsarbot:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
```

```suggestion
    runs-on: ubuntu-24.04
```

Avoid unexpected GitHub version bump.